test/jointjs/dia/Paper.js

```diff
@@ -95,6 +95,294 @@ QUnit.module('joint.dia.Paper', function(hooks) {
+                        },
+                        'label': {
+                            'text': 'Paper'
+                        }
+                    }
+                },
+                {
+                    'type': 'standard.Path',
+                    'position': {
+                        'x': 325,
+                        'y': 300
+                    },
+                    'size': {
+                        'width': 100,
+                        'height': 40
+                    },
+                    'angle': 0,
+                    'id': 'ad464767-7004-45ff-825a-c3b093639a1e',
+                    'attrs': {
+                        'body': {
+                            'd': 'M calc(0.2 * w) 0 L calc(w) 0 calc(w) calc(h) calc(0.2 * w) calc(h) 0 calc(h / 2) Z'
+                        },
+                        'label': {
+                            'text': 'Graph'
+                        }
+                    }
+                },
+                {
+                    'type': 'standard.Path',
+                    'position': {
+                        'x': 325,
+                        'y': 100
+                    },
+                    'size': {
+                        'width': 100,
+                        'height': 40
+                    },
+                    'angle': 0,
+                    'id': '0ceb97d4-dd9e-473c-879d-c1e5f04d7b49',
+                    'attrs': {
+                        'body': {
+                            'd': 'M calc(0.2 * w) 0 L calc(w) 0 calc(w) calc(h) calc(0.2 * w) calc(h) 0 calc(h / 2) Z'
+                        },
+                        'label': {
+                            'text': 'getByPath'
+                        }
+                    }
+                },
+                {
+                    'type': 'standard.Path',
+                    'position': {
+                        'x': 325,
+                        'y': 50
+                    },
+                    'size': {
+                        'width': 100,
+                        'height': 40
+                    },
+                    'angle': 0,
+                    'id': '5648dd2a-d6c8-4935-8f9c-e0edcb5f2513',
+                    'attrs': {
+                        'body': {
+                            'd': 'M calc(0.2 * w) 0 L calc(w) 0 calc(w) calc(h) calc(0.2 * w) calc(h) 0 calc(h / 2) Z'
+                        },
+                        'label': {
+                            'text': 'setByPath'
+                        }
+                    }
+                }
+            ]);
+
+            paper = new Paper({
+                el: paperEl,
+                model: testGraph,
+                async: false
+            });
+        });
+
+        QUnit.test('transformToFitContent()', function(assert) {
+
+            paper.transformToFitContent();
+
+            assert.deepEqual(paper.scale(), { sx: 1.6842105263157894, sy: 1.6842105263157894 }, 'default transform scale');
```

Comment:
Isn't that prone to rounding error? Does every browser return the same?

Reply:
It depends on the browser but it is consistent in the current test environment. I can round values but in the current implementation we will see if the browser implementation will change, i don't think it happens often.